update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]
+
+# Description:
+#   An update bash script for engelsystem
+
+# Add upstream 
+git remote add upstream http://www.github.com/fossasia/engelsystem.git
+
+# Pull the updates
+git pull upstream development
```

Comment:
Sorry. I'll change the script to my repo in this PR. And make another PR after it is merged for changing the script to pull from fossasia repo.